configuration/exodus-pipeline.yaml

```diff
@@ -438,10 +477,113 @@ Resources:
       TimeoutInMinutes: 10
     DependsOn: CodeBuildRole
 
+  IntegrationTestsCodeBuildProject:
+    Type: AWS::CodeBuild::Project
+    Condition: IsDev
+    Properties:
+      Name: !Sub ${project}-integration-tests
+      ServiceRole: !GetAtt CodeBuildRole.Arn
+      Environment:
+        Type: LINUX_CONTAINER
+        ComputeType: BUILD_GENERAL1_SMALL
+        Image: aws/codebuild/amazonlinux2-x86_64-standard:3.0
+      Source:
+        Type: CODEPIPELINE
+        BuildSpec: |
+          version: 0.2
+          phases:
+            install:
+              runtime-versions:
+                python: 3.7
+              commands:
+                - pip install --upgrade pip
+                - pip install tox
+            build:
+              commands:
+                - git clone https://github.com/$REPO_OWNER/$REPO_NAME.git -b $REPO_BRANCH
+                - cd ./$REPO_NAME
+                - virtualenv integration_tests
+                - source integration_tests/bin/activate
+                - tox -e py37 ./tests/integration -- --cdn-test-url https://$(aws cloudformation describe-stacks --stack-name $STACK_NAME | jq -r '.Stacks[0].Outputs[] | select(.OutputKey=="Distribution").OutputValue')
+      Artifacts:
+        Type: CODEPIPELINE
+      TimeoutInMinutes: 15
+
+  RollbackLambdasCodeBuild:
+    Type: AWS::CodeBuild::Project
+    # Create CodeBuild project for running integration tests against the dev pipeline
+    Condition: NotDev
```

Comment:
So, as I suggested earlier, we should probably run integration tests for dev and therefore enable rollback for it too.